microsoftgraph/microsoft-graph-devx-api
# Fix: Add work item subarea

## Overview
Fixes #1585 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Go to the knownissues endpoint
Notice that work items now have
